package.json

```diff
@@ -37,10 +35,13 @@
   },
   "devDependencies": {
     "eslint": "^5.12.1",
+    "eslint-config-prettier": "4.0.0",
+    "eslint-formatter-pretty": "^2.1.1",
     "eslint-plugin-prettier": "^3.0.1",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^1.16.3",
+    "eslint-plugin-jest": "^22.2.2",
     "jest": "^24.0.0",
+    "jest-runner-eslint": "0.7.3",
```

Comment:
Why do you use `^` for some dependencies, but exact match for others?

Reply:
Sorry, I copied them from a project of mine. Will use `^`, thanks for spotting that